packages/electrode-react-webapp/lib/react/token-handlers.js

```diff
@@ -71,14 +72,25 @@ module.exports = function setup(handlerContext /*, asyncTemplate*/) {
       : `${prodBundleBase}${assets.manifest}`;
   };
 
+  const getBundleJsNameByQuery = data => {
+    let { name } = data.jsChunk;
+    const { __dist } = data.query;
```

Comment:
so do we need to set the `_dist` in query parameter to match the babel target  ?

Reply:
This is a temporary way to explicitly use specific bundle for debug the feature. Finally, it will depends on how you want to detect the bundle you want to use